python/ray/serve/tests/test_http_cancellation.py

```diff
@@ -96,5 +99,103 @@ async def __call__(self, *args):
         assert h.remote().result() == i
 
 
+async def test_request_cancelled_error_on_http_client_disconnect_during_execution(
```

Comment:
You will need this decorator for an async test to run properly

```suggestion
@pytest.mark.asyncio
async def test_request_cancelled_error_on_http_client_disconnect_during_execution(
```